app/models/benefit_model.rb

```diff
@@ -5,6 +5,51 @@ class BenefitModel
 
   FREQUENCY_OPTIONS = %w[every_week every_two_weeks every_four_weeks monthly].freeze
 
+  PASSPORTED_BENEFITS = [
+    "Universal Credit",
+    "Income Support",
+    "Income-based Jobseeker's Allowance (JSA)",
```

Comment:
CFE doesn't return this, but it does return `Jobseeker’s Allowance (JSA)`. Likewise it returns `Employment and Support Allowance (ESA)` and `Pension Credit`. Do we want to remove those elements from the list? If so we should modify those strings in this array. If not, we should remove these strings.